perception/bytetrack/README.md

```diff
@@ -12,6 +12,8 @@ the number of false negatives is expected to decrease by using it.
 
 ### Cite
 
+<!-- cspell: ignore Yifu Peize Jiang Dongdong Fucheng Weng Zehuan Xinggang -->
```

Comment:
FYI: There is a dictionary for people names.
https://github.com/tier4/cspell-dicts/tree/main/people-names

Reply:
Thank your for giving me infomation!
These names are included in bytetrack package only, so I think it is good to add cspell ignore instead of adding authors' name to people-names dictionary.

Reply:
Either one is okay for me.